packages/design-system/src/components/N8nInputNumber/InputNumber.vue

```diff
@@ -1,18 +1,21 @@
-<script lang="ts">
+<script setup lang="ts">
+import type { InputSize } from '@/types';
 import { ElInputNumber } from 'element-plus';
-import { defineComponent } from 'vue';
+import { computed } from 'vue';
 
-export default defineComponent({
-	name: 'N8nInputNumber',
-	components: {
-		ElInputNumber,
-	},
-	props: {
-		...ElInputNumber.props,
-	},
-});
+type InputNumberProps = {
+	size?: InputSize;
+	min?: number;
+	max?: number;
+	step?: number;
+	precision?: number;
+};
+
+const props = defineProps<InputNumberProps>();
+
+const elSize = computed(() => props.size as 'small' | 'large');
```

Comment:
Just a nitpick: this may not be a computed, props are reactive to and we usually don't define size as a dynamic variable anyways.

Reply:
I think I was doing something with it before and forgot to clean it up, removed the computed